tvb_nest/tests/test_reduced_wong_wang_exc_io_inh_i.py

```diff
@@ -1,30 +1,30 @@
 # -*- coding: utf-8 -*-
 
 import os
-from oct2py import octave
```

Comment:
After discussing and understanding the previous unit-test we decided it is best to keep the octave dependency, enhance the dockerimage to include octave and oct2py python library and run the original test

Reply:
Ok. The suggestion has been applied and it can be seen in a future commit. In this case, the file rww_dfun.py, added at the latest commit, should be ignored at the merge.